apm-lambda-extension/main_test.go

```diff
@@ -345,13 +344,14 @@ func sendLogEvent(requestId string, logEventType logsapi.SubEventType) {
 		extension.Log.Errorf("Could not encode record : %v", err)
 		return
 	}
-	host, port, _ := net.SplitHostPort(logsapi.TestListenerAddr.String())
+	// TODO refactor these tests
```

Comment:
Did you say you were going to skip the tests until this was done? The tests in the main package are failing at the moment because of this.

Reply:
Done: https://github.com/elastic/apm-aws-lambda/issues/249